libs/common/random.go

```diff
@@ -175,146 +166,95 @@ MAIN_LOOP:
 	return string(chars)
 }
 
-// It is not safe for cryptographic usage.
```

Comment:
We should air on the side of caution with warnings like this. Better to keep them than remove!

Reply:
They were moved to the `Rand` struct documentation (not removed). https://github.com/tendermint/tendermint/blob/948b91e62ec2d46825fde843d0403e4fcc4510fb/libs/common/random.go#L14-L20